package.json

```diff
@@ -30,7 +31,8 @@
         "format-check": "prettier --config prettier.config.js --check \"**/*\"",
         "prepack": "npm run format-check && npm run build",
         "watch:build": "tsc -p . --watch",
-        "watch:test": "jest --watch --coverage false --colors"
+        "watch:test": "jest --watch --coverage false --colors",
+        "copyrightheaders": "license-check-and-add"
```

Comment:
Can you add this into prepack like we've done for format-check (and lint, in #7)?

Reply:
Updated.